helm/chart/router/README.md

```diff
@@ -2,7 +2,7 @@
 
```

Comment:
Hmm, that's odd. The version hasn't been updated to 0.1.22 in the Chart.yaml, but the version has been updated in README.md. Was `helm-docs` used to generate these docs or were they manually edited? It's important to run `helm-docs` or we won't get the correct information in the README.md since it won't pick up template changes.

Reply:
i definitely ran it.

Reply:
maybe i staged it wrong? or maybe the PR i merged with the helm changes undid it?

Reply:
I imagine that's it. It must have overwritten the changes you had made.